src/engine/N3Base/N3FXPartMesh.cpp

```diff
@@ -228,6 +228,10 @@ bool CN3FXPartMesh::Load(HANDLE hFile) {
         ReadFile(hFile, &m_vUnitScale, sizeof(__Vector3), &dwRWC, NULL);
     }
 
+    if (m_iVersion == 6) {
```

Comment:
Actually there is I got this all working already and 1298 fx loading up in other repo/branch where I completed this before ragequiting due to stress on snoxdko discord. About the other if m version 3 is because I intended it to be last major version to be supported and that's why it was ==3. For this one I can't comment much on  it have to double triple check my earlier work.

Reply:
No I get that, I'm just saying that there's really no downside to just swapping that to `>=`. All it does is it futureproofs it with no real downside.

Reply:
That's true

Reply:
Mgame has a bad habit of doing things weirdly. I've come across parameters that were removed or changed to a different position that's why I probably did this it's hard to tell without the src I had on a diff branch/repo where it was working as in atleast loading and displaying it properly for 95%. Just sometimes scaling or newer functions that aren't added yet but atleast it solved all the crashing.

Reply:
But you are probably right anyway. Hey were talking to 2stars afteral. At this point it was all a WIP and I don't know asm so I had to brainstorm to come so far.